test/bdd/features/release.feature

```diff
@@ -7,21 +7,15 @@ Feature: Release related tests
   Scenario: Node is able to publish on the network, expect status completed on publish result
     Given I setup 4 nodes
     When I call publish on node 1 with validAssertion
-    And I wait for last publish to finalize
     And Last publish finished with status: COMPLETED
-
-#  @release
-#  Scenario: Node is able to resolve assertion previously published
-#    Given I setup 4 nodes
-#    When I call publish on node 1 with validAssertion
-#    And I wait for last publish to finalize
-#    And Last publish finished with status: COMPLETED
-#    And I call resolve on node 1 for last published assertion
-#    And I wait for last resolve to finalize
-#    And Last resolve finished with status: COMPLETED
-#    And Last resolve returned valid result
-##    And I setup 1 additional node
-##    And I call resolve on node 5 for last published assertion
-##    And I wait for last resolve to finalize
-##    And Last resolve finished with status: COMPLETED
-##    And Last resolve returned valid result
+#
+  @release
```

Comment:
Are you sure that this test will be executed since we changed command for starting the tests?

Reply:
I see that this step was executed in gihub action so just ignore this comment :-)